Modules/_testcapimodule.c

```diff
@@ -7183,6 +7200,12 @@ PyInit__testcapi(void)
     }
     PyModule_AddObject(m, "HeapDocCType", HeapDocCType);
 
+    PyObject *NullTpDocType = PyType_FromSpec(&NullTpDocType_spec);
```

Comment:
Please add a comment starting with "bpo-41832: " to explain the purpose of this type (test NULL tp_doc).

Reply:
done.